docs/bnb_smart_chain/staking/index.md

```diff
@@ -0,0 +1,191 @@
+# BSC Staking Overview
+
+BNB Smart Chain (BSC) is a Proof-of-Staked-Authority (PoSA) blockchain where staking plays a crucial role.
+[BEP-294](https://github.com/bnb-chain/BEPs/pull/294) introduces native staking on BSC.
+This section covers the fundamental staking concepts and operations on BSC.
+
+## Basic Concepts
+
+### Consensus Engine
+
+BSC uses a consensus mechanism which combines DPoS and PoA for consensus, so that:
+
+* Blocks are produced by a limited set of validators.
+* Validators take turns to produce blocks in a PoA manner.
+* Validator set are elected in and out based on a staking based governance.
+
+To determinate which validators are eligible to produce blocks, that's why staking mechanism is needed.
+
+### Validator Set
+
+The validator set is the group of nodes that are responsible for validating transactions and producing blocks on the
+BSC. The validator set is determined by the amount of staking credit each validator has, which reflects the amount of
+BNB staked by the validator and its delegators. The top validators with the most staking credit are selected as the
+active validator set, and they take turns to propose and vote on blocks. The rest of the validators are in the standby
+validator set, and they can join the active validator set if their staking credit increases or if some active validators
+drop out.
+
+Validators can join or leave the validator set by creating or editing their validator information.
+Validators can also be removed from the validator set by slashing, which is a penalty for misbehaving or being offline.
+
+### Validator Election
+
+There are different roles for validators:
+
+* Cabinet: the top K (which is 21 currently) validators who get the most chance of producing blocks.
+* Candidate: the top (K, K+NumOfCandidates] validators who get a small chance of producing blocks.
```

Comment:
==> `Candidate: the top (K, K+NumOfCandidates] (which is (21,45] currently) validators who get a small chance of producing blocks.`

Reply:
fixed